src/agent/agent.c

```diff
@@ -1492,6 +1575,69 @@ bool agent_start(Agent *agent) {
                 sd_bus_add_filter(agent->systemd_dbus, NULL, debug_systemd_message_handler, agent);
         }
 
+<<<<<<< HEAD
+=======
```

Comment:
some leftover merge stuff here

Reply:
I think this was due the recent merge from martins config PR. Resolved it.